### PR TITLE
doc: fixed architecture link in get-started.md

### PR DESCRIPTION
**Pull request by dotdc**
Signed-off-by: David <davidcalvertfr@gmail.com>

### Description:

- fixed architecture link in get-started.md

### Related Issue:

Fixes: https://github.com/vdaas/vald/issues/1618

### How Has This Been Tested?:

Not tested, just a doc change

### Environment:

- Go Version: 1.18
- Docker Version: 20.10.8
- Kubernetes Version: 1.22.0
- NGT Version: 1.14.3

### Types of changes:

- [x] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [x] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [x] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/tutorial/get-started.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ And, Fashion-mnist is used as an example of a dataset.
 
 The below image shows the architecture image about the deployment result of `Get Started`.<br>
 The 4 kinds of components, `Vald LB Gateway`, `Vald Discoverer`, `Vald Agent`, and `Vald Index Manager` will be deployed to the Kubernetes.
-For more information about Vald's architecture, please refer to [Architecture](../../docs/overview/architecture.md).
+For more information about Vald's architecture, please refer to [Architecture](../../overview/architecture.md).
```
Comment (vankichi) [inline docs/tutorial/get-started.md:13]:
```suggestion
For more information about Vald's architecture, please refer to [Architecture](../overview/architecture.md).
```
It is the correct path :)

**Review by vankichi — COMMENTED** [review]
Hi @dotdc 

Thanks for PR !
I just commented about the document path.
Please check 😄

**Comment by dotdc** [comment]
Hi @vankichi,
Just fixed the path like you mentioned in the comment.
Let me know 🙂

**Review by vankichi — APPROVED** [review]
LGTM

**Comment by kpango** [comment]
@all-contributors please add @docdc for doc

**Comment by kpango** [comment]
https://github.com/all-contributors please add @dotdc for doc

**Comment by kpango** [comment]
@all-contributors please add @dotdc for doc

**Review by kpango — APPROVED** [review]

**Pull request merged**